### PR TITLE
adding theatre controller

### PR DESCRIPTION
**Pull request by RashmiNayak-S**

### CODE REVIEW

File: src/BookYourShow.Api/Controllers/TheatreController.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        #region get theatres
+        [HttpGet]
+        [ProducesResponseType(typeof(Theatre), 200)]
+        [ProducesResponseType(404)]
+        
+        public async Task<IActionResult> GetTheatre()
+        {
+
+            var theatres = await theatreRepository.GetTheatre();
+            //throw new Exception("Exception occurred!");
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/TheatreController.cs]: remove

File: src/BookYourShow.Api/Controllers/TheatreController.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            if (theatres == null)
+            {
+                return NotFound();
+            }
+            return Ok(theatres);
+
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/TheatreController.cs]: remove unwanted spaces

File: src/BookYourShow.Api/Controllers/TheatreController.cs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        #endregion
+
+        #region update Theatre
+
+        [HttpPut]
+        [ProducesResponseType(typeof(Theatre), 200)]
+        [ProducesResponseType(404)]
+        [ProducesResponseType(400)]
+        
+        //[Route("updatetheatre")]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/TheatreController.cs]: ?

**Review by basilin — CHANGES_REQUESTED** [review]

File: src/BookYourShow.Api/Controllers/TheatreController.cs (hunk 1 above)
Comment (RashmiNayak-S) [reply src/BookYourShow.Api/Controllers/TheatreController.cs]: resolved

File: src/BookYourShow.Api/Controllers/TheatreController.cs (hunk 2 above)
Comment (RashmiNayak-S) [reply src/BookYourShow.Api/Controllers/TheatreController.cs]: resolved

File: src/BookYourShow.Api/Controllers/TheatreController.cs (hunk 3 above)
Comment (RashmiNayak-S) [reply src/BookYourShow.Api/Controllers/TheatreController.cs]: resolved

**Pull request merged**